pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - "typescript/sdk"
```

Comment:
why is this only limited to only these 3 pkgs?
shouldnt everything under the typescript dir be included??

Reply:
couldnt you do `typescript/*`